### ISSUE TITLE
Public PCs

### ISSUE THREAD

**Issue by salomerohr** [CERNDocumentServer/cds-ils#836]
**Is your feature request related to a problem? Please describe.**
There are 2 public PCs in the library for people to consult the library catalogue. We would like the PCs to stay on the library catalogue but users easily get out by consulting external links in the catalogue. 

**Describe the solution you'd like**
Would there be any solution to help with that?

**Comment by kpsherva** [comment]
agreed to be closed since it refers to local PC configuration, which is not the part of CERN Library Catalogue application
